tests/integration/component.go

```diff
@@ -215,10 +208,9 @@ func componentTests(args ...string) {
 			Expect(cmpList).To(ContainSubstring("cmp-git-2"))
 			Expect(cmpList).To(ContainSubstring("Not Pushed"))
 			Expect(cmpList).To(ContainSubstring("Pushed"))
-
-			helper.CmdShouldPass("odo", append(args, "delete", "-f", "--all", "--context", context)...)
```

Comment:
It's a verification step of how component delete works with ```--all``` flag when you have multiple component in same app but in different context. This is a useful user scenario, so it can not be removed.

Add some verification step to confirm component delete with ```--all``` flag works even in this scenario